edxval/models.py

```diff
@@ -378,13 +378,20 @@ class for providing tuple choices.
     CUSTOM = 'Custom'
     THREE_PLAY_MEDIA = '3PlayMedia'
     CIELO24 = 'Cielo24'
+    EDX_AI_TRANSLATIONS = 'edx_ai_translations'
 
     CHOICES = (
         (CUSTOM, CUSTOM),
         (THREE_PLAY_MEDIA, THREE_PLAY_MEDIA),
         (CIELO24, CIELO24),
     )
 
+    # Choices specififcally for only the VideoTranscript model
```

Comment:
Isn't the `CHOICES` enumeration already isolated to the VideoTranscript model?

Reply:
TranscriptPreference and ThirdPartyTranscriptCredentialsState also use this model